pytentiostat/main.py

```diff
@@ -3,19 +3,65 @@
 from pytentiostat.reporter import save_data_to_file
 from pytentiostat.tester import experiment
 from pytentiostat.routines import startup_routine, closing_routine
+from pytentiostat.plotter import plot_updater, plot_initializer
+import matplotlib.pyplot as plt
+import sys
+import time
 
-config_data = parse_config_file()
 
-# Initialize Experiment
-com, board, a0, a2, d9 = startup_routine()
+class BoardCom:
+    def __init__(self):
+        self.com, self.board, self.pin_a0, self.pin_a2, self.pin_d9 = startup_routine()
```

Comment:
it strikes me that if we are going to do this it would make sense to  make ``startup_routine()`` a method of the BoardCom class?

Reply:
Yes, that would make it more concise. Maybe I should move the class definition into routines.py and we can import it into main.